core-ui/src/components/Predefined/Details/Roles/Rules.js

```diff
@@ -1,45 +1,76 @@
 import React from 'react';
-import { LayoutPanel } from 'fundamental-react';
+import { GenericList, EMPTY_TEXT_PLACEHOLDER } from 'react-shared';
 import { useTranslation } from 'react-i18next';
+import { Icon } from 'fundamental-react';
 
-import './Rules.scss';
+function VerbStatus({ rule, verb }) {
+  const hasVerb = rule.verbs?.includes(verb) || rule.verbs?.includes('*');
+
+  return hasVerb ? <Icon glyph="accept" ariaHidden /> : null;
```

Comment:
Maybe we should add an empty placeholder as well instead of null?

Reply:
I was thinking about it, but I was a little bit concerned that it would be too cluttered. When I try it out now, it actually... looks fine.